gammapy/data/event_list.py

```diff
@@ -132,7 +132,14 @@ def altaz(self):
 
         lon, lat = self['AZ'], self['ALT']
         return SkyCoord(lon, lat, unit='deg', frame=altaz_frame)
-
+    
+    @property
+    def offset(self):
```

Comment:
I see roughly the same offset (differences at the 0.01 deg level), no matter if it's computed for RA/DEC or DETX/DETY:
```python
def print_offset(filename):
    import numpy as np
    from gammapy.datasets import gammapy_extra
    from gammapy.data import EventList

    filename = gammapy_extra.filename(filename)
    event_list = EventList.read(filename, hdu='EVENTS')
    theta1 = event_list.pointing_radec.separation(event_list.radec).deg
    theta2 = np.sqrt(event_list['DETX'].data ** 2 + event_list['DETY'].data ** 2)
    print(theta1[:5])
    print(theta2[:5])
```
```python
>>> print_offset('test_datasets/unbundled/hess/run_0023037_hard_eventlist.fits.gz')
[ 1.90449774  0.49800608  1.44632578  1.19342053  1.37034893]
[ 1.89962924  0.49339229  1.45202422  1.19860065  1.37333238]
>>> print_offset('datasets/hess-crab4/hess_events_simulated_023523.fits')
[ 0.54917634  0.50370294  0.55434477  0.52719957  0.62944549]
[ 0.54917902  0.50370306  0.55434608  0.52720046  0.62944645]
```

I think it's OK if you just put the sky separation for this pull request to finish it up,
no need to trace back the difference you saw.